packages/tackle-box/lib/components/Text/Text.tsx

```diff
@@ -6,40 +6,33 @@ type Sizes = "4xl" | "3xl" | "2xl" | "xl" | "lg" | "md" | "sm" | "xs";
 const styles = css.create({
   base: {
     fontFamily: "Favorit",
+    textAlign: "center",
```

Comment:
Does this not get overwritten by the `textAlign` prop that we default to `left`?

Reply:
Good catch 👍